custom-tests.yaml

```
@@ -1505,10 +1505,15 @@ api:
       if (!constructor) {
         return false;
       }
+
+      // Earlier versions of Firefox expose the "RTCPeerConnection" variable
```

```suggestion
      // Earlier versions of Firefox expose mozRTCPeerConnection 
```

Was it `mozRTCPeerConnection` at that time? Trying to avoid the word "variable" here since it's rather a property on the global object.